db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.1].define(version: 2024_05_09_150727) do
+ActiveRecord::Schema[7.1].define(version: 2024_05_20_090646) do
```

Comment:
Hi @TBenedyk the timestamp in the version is not matching the timestamp from the new migration file added `db/migrate/20240515145620_set_lead_provider_names.rb` .. could you please double check? tkss

Reply:
Hi @leandroalemao this has been updated now. Are you able to take a look? Thanks!